public/app/features/provisioning/GettingStarted/GettingStarted.tsx

```diff
@@ -0,0 +1,270 @@
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+import { LinkButton, Alert, Stack, Text, Button, Box, Icon } from '@grafana/ui';
+
+import { useGetFrontendSettingsQuery } from '../api';
+import { CONNECT_URL, MIGRATE_URL } from '../constants';
+import { css } from '@emotion/css';
+import { getConfigurationStatus } from './utils';
+import { useState } from 'react';
+import { SetupModal } from './SetupModal';
+import { feature_ini, ngrok_example, root_url_ini } from './types';
```

Comment:
All the FE code should use camel case. Also these don't look like types, so shouldn't be in the `types` file.